lib/omniauth/strategies/stackexchange.rb

```diff
@@ -39,7 +38,7 @@ def raw_info
         @raw_info ||= access_token.get('me', :params => params).parsed['items'].first
 
         unless @raw_info
-          raise NotRegisteredForStackExchangeSiteError, "User is not registered for requested StackExchange site (#{site})"
+          raise OAuth2::CallbackError.new("User is not registered for requested StackExchange site (#{site})")
```

Comment:
I think there's no need for creating a new object here, you can use it like this: 

```ruby
raise OAuth2::CallbackError, "User is not registered for requested StackExchange site (#{site})"
```